vector/src/main/res/values/strings.xml

```diff
@@ -2523,9 +2523,9 @@
     <string name="ftue_auth_carousel_1_title">Own your conversations.</string>
     <string name="ftue_auth_carousel_1_body">End-to-end encrypted messaging for secure and independent communication, connected via Matrix.</string>
     <string name="ftue_auth_carousel_2_title">You\'re in control.</string>
-    <string name="ftue_auth_carousel_2_body">Element lets you choose where you messages are stored, keeping you in control of your data.</string>
-    <string name="ftue_auth_carousel_3_title">Connect with anyone.</string>
-    <string name="ftue_auth_carousel_3_body">Element works with all Matrix-based apps and can even bridge into proprietary messengers.</string>
+    <string name="ftue_auth_carousel_2_body">Element lets you choose where your messages are stored, keeping you in control of your data.</string>
```

Comment:
We should not use hard-coded `Element` in the string. Use `{app_name}` and add `template` to the string key. (see other example in this file.

Reply:
https://github.com/vector-im/element-android/pull/4880/commits/57ccf2d16a115e47809e87dc63aac682dd45fc1e

also marks the previous strings for removal